elk.yml

```diff
@@ -1,44 +1,52 @@
 # vim: ft=ansible
 ---
 - hosts: elasticsearch
+  become: true
   tags:
     - elasticsearch
     - elk
 
   vars_files:
-    - "{{ inventory_dir  }}/../vars/main.yml"
+    - "{{ inventory_dir  }}/{{ vars_files_relative }}/vars/main.yml"
 
   roles:
     - { role: 'geerlingguy.repo-epel' }
     - { role: 'leifmadsen.elasticsearch' }
 
 - hosts: logstash
+  become: true
   tags:
     - logstash
     - elk
 
   vars_files:
-    - "{{ inventory_dir  }}/../vars/main.yml"
+    - "{{ inventory_dir  }}/{{ vars_files_relative }}/vars/main.yml"
 
   roles:
     - { role: 'geerlingguy.repo-epel' }
     - { role: 'geerlingguy.java' }
     - { role: 'leifmadsen.logstash' }
 
 - hosts: kibana
+  become: true
   tags:
     - kibana
     - elk
 
   vars_files:
-    - "{{ inventory_dir  }}/../vars/main.yml"
+    - "{{ inventory_dir  }}/{{ vars_files_relative }}/vars/main.yml"
 
   roles:
     - { role: 'geerlingguy.repo-epel' }
     - { role: 'geerlingguy.nginx' }
     - { role: 'leifmadsen.kibana-4' }
 
   post_tasks:
+    - name: Ensure libselinux-python is installed
+      yum:
+        name: libsemanage-python
```

Comment:
Could you please change "libselinux-python" to "libsemanage-python" for consistency?